### PR TITLE
Add Mainnet tomls for SIP-373 and SIP-376

### PR DESCRIPTION
**Pull request by leomassazza**

### CODE REVIEW

File: tomls/omnibus-base-mainnet-andromeda/perps/arb-invokes.toml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+include = [
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/arb-invokes.toml]: please make sure to remove any nested includes and put them on the top level.
I have done it for testnet and they somehow appeared here for mainnet too...

File: package.json [hunk 2]
```diff
@@ -39,8 +39,8 @@
   },
   "devDependencies": {
     "@pythnetwork/pyth-evm-js": "^1.33.0",
-    "@usecannon/builder": "2.12.2",
-    "@usecannon/cli": "2.12.2",
+    "@usecannon/builder": "2.12.3",
```
Comment (noisekit) [inline package.json:42]: Cannon update is better in a separate PR

File: tomls/omnibus-base-mainnet-andromeda/perps/arb-invokes.toml [hunk 3]
```diff
@@ -0,0 +1,149 @@
+[setting.perpsArbMarketId]
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/arb-invokes.toml:1]: I believe we should have a "package" "standard-perps-market" and provision it with all settings in one simple step. As all of the standard markets are copypasted now

**Review by noisekit — APPROVED** [review]

File: tomls/omnibus-base-mainnet-andromeda/perps/btc-invokes.toml [hunk 4]
```diff
@@ -1,5 +1,3 @@
-include = ["../../oracles/btc.toml", "../../markets/common/bigcap-settings.toml"]
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/btc-invokes.toml:1]: wrong oracle was here

**Comment by kaleb-keny** [comment]
awesome all tests green for today's release, thank you @noisekit @leomassazza

**Pull request merged**